src/components/ListItem.jsx

```diff
@@ -1,5 +1,41 @@
+import React, { useState, useEffect } from 'react';
 import './ListItem.css';
+import { updateItem } from '../api/firebase';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ item, listPath }) {
+	const { id, totalPurchases, name, dateLastPurchased } = item;
+	const lessThan24HoursSincePurchase =
+		dateLastPurchased &&
+		(new Date() - dateLastPurchased.toDate()) / 3600000 < 24;
```

Comment:
nit: I saw Aditya mentioned this in the other PR. While this is correct there are two things to consider:

- Readability
- Reusability

For items like this I usually move this into a function which meets both of the above.

So an example function could be: `compareIfDateIsLessThan24Hours` and your const should usually have a prefix like `is`, `has`. So `isLessThan24HoursSincePurchase` would be more clear this is a boolean.

Small nits but wanted to put a comment here.